resources/views/dinnerform/admin_includes/buttonbar.blade.php

```diff
@@ -0,0 +1,14 @@
+<div class="card-footer border-bottom">
```

Comment:
Kan deze niet bij `resources/views/dinnerform/dinnerform_block.blade.php`? En dan afhankelijk van de rechten en waar hij getoont wordt knoppen wel niet meegeven?

Reply:
Is there a reason you resolved this comment but didn't do anything with the question/suggestion?